code/datums/brain_damage/special.dm

```diff
@@ -15,7 +15,7 @@
 	if(prob(4))
 		if(prob(33) && (owner.IsStun() || owner.IsKnockdown() || owner.IsUnconscious()))
 			speak("unstun", TRUE)
-		else if(prob(60) && owner.health <= HEALTH_THRESHOLD_CRIT)
```

Comment:
Why not just use a is_in_crit() proc or a define or something similar? You copy and paste the `+ owner.crit_modifier()` quite a lot

Reply:
proc is overhead

Reply:
\>calls procs overhead
\>adds his own proc to calculate modifiers
scumbag steve

Reply:
![image](https://user-images.githubusercontent.com/20987591/42142582-8044e1c8-7db0-11e8-9ec6-1d0069229010.png)
you ignored me after that :(

Reply:
how come it dosn't show my owner status bad

Reply:
u r a sick sick girl.